### ISSUE TITLE
php ds dependency

### ISSUE THREAD

**Issue by in3rsha** [neo4j-php/neo4j-php-client#81]
**Library version: 2.1.1**

php-ds was removed as a dependency in 5e36e22c65157142dfc4d6f044fe374711167245, but I think it may still be required.

I have done a fresh `composer update` with the latest version (2.1.1) and I get this error:

```
PHP Fatal error:  Uncaught Error: Class "Ds\Map" not found in vendor/laudis/neo4j-php-client/src/ClientBuilder.php:67
Stack trace:
#0 file.php(1): Laudis\Neo4j\ClientBuilder::create()
```

**Comment by transistive** [comment]
You are absolutely correct, it appears my search and replace didn't work correctly. I'll fix it now!

**Comment by transistive** [comment]
Fixed in version 2.1.2 and commits #7eadbc18969efad47f246ed98ec235212f7031a2 and #db90921fc65ffaa550d7fbd26c37bb8a32bb98de
